src/ScaleUnitManagement/UserConfig.sample.xml

```diff
@@ -23,6 +23,12 @@
     <!-- If empty, the existing value in web.config will be used - otherwise the web.config will get overwritten with this value. -->
     <AADTenantId></AADTenantId>
   </AADConfiguration>
+
+  <ADFSConfiguration>
```

Comment:
Please add a comment similar to that of the `AADConfiguration` element, indicating that this is only considered for the `LBD` environment type and that the appid is used to communicate with the environment from the tool.

Reply:
Also add to the XML comment at line 43 details about what the LBD enum value is for (similar to existing documentation).